pymc/model_graph.py

```diff
@@ -156,6 +213,29 @@ def update_node_formatters(node_formatters: NodeTypeFormatterMapping) -> NodeTyp
     return node_formatters
 
 
+class AddNode(Protocol):
```

Comment:
Why this?

Reply:
mypy to allow callable with first arg str then any amount of kwargs. 
Do you have an alternative?

Reply:
# type: ignore